ConwaysGameOfLife.Tests/GameControllerShould.cs

```diff
@@ -18,128 +18,150 @@ public GameControllerShould()
+            var input = "q";
             _input.GetReadLine(input);
-            _controller.CreateInitialUniverse();
 
-            _controller.DisplayUniverse();
-            var actual = _output.GetLastWriteLine();
+            _controller.Run();
 
-            Assert.Equal(expected, actual);
+            Assert.Equal(expectedOutput, _output.GetLastWriteLine());
         }
 
         [Theory]
-        [InlineData("3,3")]
-        [InlineData("10,10")]
-        public void ReturnUniverseString_GivenValidInput(string input)
+        [InlineData("3,3", "💀💀💀\n💀💀💀\n💀💀💀\n")]
+        [InlineData("3,2", "💀💀💀\n💀💀💀\n")]
+        [InlineData("9,9", "💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n")]
+        [InlineData("3,4", "💀💀💀\n💀💀💀\n💀💀💀\n💀💀💀\n")]
+        public void DisplayUniverse_GivenValidUniverseString(string input, string expected)
         {
             _input.GetReadLine(input);
-            var expected = input;
-
-            var result = _controller.CreateValidUniverseString();
+            _input.GetReadLine("q");
+            
+            _controller.Run();
+            var actual = _output.GetLastWriteLine();
 
-            Assert.Equal(expected, result);
+            Assert.Equal(expected, actual);
         }
 
         [Theory]
-        [InlineData(new string[] { "0,3", "3,0" }, "1,1")]
-        [InlineData(new string[] { "0;3", "3-0", "4 4" }, "3,3")]
-        [InlineData(new string[] { ",," }, "10,10")]
-        [InlineData(new string[] { "3,3," }, "10,10")]
-        [InlineData(new string[] { "3,3,3", "3;3;" }, "3,3")]
-        [InlineData(new string[] { "a,b" }, "3,3")]
-        [InlineData(new string[] { "-1,4", "4,-1" }, "3,3")]
-        [InlineData(new string[] { "10,0", "0,10" }, "3,3")]
-        public void ReturnUniverseString_GivenInvalidInputsFollowedByValidInput(IEnumerable<string> invalidInputs, string validInput)
+        [InlineData(new string[] { "0,3", "3,0" }, "3,3", "💀💀💀\n💀💀💀\n💀💀💀\n")]
+        [InlineData(new string[] { "0;3", "3-0", "4 4" }, "3,3", "💀💀💀\n💀💀💀\n💀💀💀\n")]
+        [InlineData(new string[] { ",," }, "9,9", "💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n")]
+        [InlineData(new string[] { "3,3," }, "9,9", "💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n💀💀💀💀💀💀💀💀💀\n")]
+        [InlineData(new string[] { "3,3,3", "3;3;" }, "3,3", "💀💀💀\n💀💀💀\n💀💀💀\n")]
+        [InlineData(new string[] { "a,b" }, "3,3", "💀💀💀\n💀💀💀\n💀💀💀\n")]
+        [InlineData(new string[] { "-1,4", "4,-1" }, "3,3", "💀💀💀\n💀💀💀\n💀💀💀\n")]
+        [InlineData(new string[] { "10,0", "0,10" }, "3,3", "💀💀💀\n💀💀💀\n💀💀💀\n")]
+        public void DisplayUniverse_GivenInvalidInputsFollowedByValidInput(IEnumerable<string> invalidInputs, string validInput, string expected)

```

Comment:
```suggestion
        public void CreateUniverse_GivenInvalidInputsFollowedByValidInput(IEnumerable<string> invalidInputs, string validInput, string expected)

```

Reply:
What is tested here is that the universe was created successfully from the valid input at the end, and the displaying of the universe is more of how it's tested